packages/manager/modules/pci/src/projects/project/data-processing/jobs/data-processing-jobs.html

```diff
@@ -0,0 +1,124 @@
+<div data-ui-view>
+    <div>
+        <!-- cui messages -->
+        <cui-message-container
+            data-messages="$ctrl.messages"
+        ></cui-message-container>
+        <oui-datagrid
+            data-rows-loader="$ctrl.getJobs($config)"
+            data-refreshable
+            criteria="$ctrl.criteria"
```

Comment:
I think "data-criteria" is better and you can change below with "data-on-criteria-change".